vm/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -3,75 +3,57 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-VM Connector v2.0
+Anypoint Connector for Virtual Machine (VM Connector) handles intra-app and inter-app communication through asynchronous queues that can be transient or persistent.
 
-Anypoint Connector for Virtual Machine (VM) (VM Connector) handles intra-app and inter-app communication through asynchronous queues that can be transient or persistent.
+* Transient queues are faster than persistent queues, but they are not reliable in the case of a system crash
+* Persistent queues are slower but reliable
 
-* Transient queues are faster than persistent queues, but they are not reliable in the case of a system crash.
-* Persistent queues are slower but reliable.
+When running on a single instance, persistent queues work by serializing and storing the contents on the disk. However, when running in cluster mode, the memory grid backs the persistent queues. Thereby, when a flow uses the VM Connector to publish content to a queue, Mule runtime engine determines whether to process the message in the same origin node or to send the message to the cluster to pick another node. The VM Connector facilitates the load distribution across a cluster.
+In either case, when using persistent queues the contents you send must be serializable. Even though you can enable Kryo serialization to get a wider range of serializable values, Kryo does have some limitations. Therefore, try to:
 
-When running on a single instance, persistent queues work by serializing and storing the contents on the disk.
+* Keep your values simple because complex structures might result in serialization errors or performance issues if they are not easy to serialize
+* Make sure complex Java object implements the `Serializable` interface and that they conform to the Java Bean contract
```

Comment:
What is the subject of this sentence: object or objects? My guess...:
```suggestion
* Ensure that complex Java objects implement the `Serializable` interface and that they conform to the Java Bean contract.
```
...but does an object "implement" an interface?